configure

```diff
@@ -174,6 +174,19 @@ while true; do
 
   if [[ -z "$TF_CUDNN_VERSION" ]]; then
     TF_CUDNN_EXT=""
+    # Resolve to the SONAME of the symlink.  Use readlink without -f
+    # to resolve exactly once to the SONAME.  E.g, libcudnn.so ->
+    # libcudnn.so.4
+    REALVAL=`readlink ${CUDNN_INSTALL_PATH}/lib64/libcudnn.so`
```

Comment:
Installing CuDNN 5.0 only gave me this file on MacOS
`-rwxr-xr-x@ 1 yaroslavvb  staff  58270280 Apr 23 03:19 /usr/local/cuda/lib/libcudnn.5.dylib
`
I then manually added this symlink to make it work
`lrwxr-xr-x  1 yaroslavvb  admin        16 Jun 17 04:24 /usr/local/cuda/lib/libcudnn.dylib -> libcudnn.5.dylib
`